src/main/webapp/find-events.html

```diff
@@ -25,10 +25,10 @@
       </div>
```

Comment:
Changing `<script src="./assets/script.js"></script>` to `<script defer src="./assets/script.js"></script>` should fix the null error commented above

Reply:
My error was happening due to a different issue - accessing pages in which there is no `all-posts` element (see other comment)